manifests/Adobe/AdobeAcrobatReaderDC/2020.009.20063.yaml

```diff
@@ -0,0 +1,15 @@
+Id: Adobe.AdobeAcrobatReaderDC
+Version: 2020.009.20063
+Name: AdobeAcrobatReaderDC
+Publisher: Adobe
+License: Adobe General Terms of Use
+LicenseUrl: https://www.adobe.com/legal/terms.html
+Tags: PDF
+Description: View, sign, collaborate on, and annotate PDFs with our free Adobe Acrobat Reader. 
+Homepage: https://acrobat.adobe.com/us/en/acrobat/pdf-reader.html?promoid=TPQVLCXQ
+Installers:
+  - Arch: x86
+    Url: https://ardownload2.adobe.com/pub/adobe/reader/win/AcrobatDC/1901020064/AcroRdrDC1901020064_MUI.exe
+    Sha256: 81953F3CF426CBE9E6702D1AF7F727C59514C012D8D90BACFB012079C7DA6D23
+    InstallerType: exe
+    Language: Multilingual
```

Comment:
```suggestion
Id: Adobe.AcrobatReaderDC
Version: 2020.009.20063
Name: Adobe Acrobat Reader DC
Publisher: Adobe
License: Copyright (c) Adobe
LicenseUrl: https://www.adobe.com/content/dam/cc/en/legal/licenses-terms/pdf/Reader-EULA-en_US-20181207.pdf
AppMoniker: adobereader
Description: Only with Adobe Acrobat Reader you can view, sign, collect and track feedback, and share PDFs for free. 
Homepage: https://acrobat.adobe.com/us/en/acrobat/pdf-reader.html
Installers:
  - Arch: x86
    Url: http://ardownload.adobe.com/pub/adobe/reader/win/AcrobatDC/2000920063/AcroRdrDC2000920063_en_US.exe
    Sha256: B49F6B49F28712735098836BB7D9BF2FF12568ED166936B7B90C87BC847B2C68
    InstallerType: inno
```

Reply:
I will go and check weather the AcroRdrDC2000920063_en_US.exe is the newest version.

Reply:
I found that AcroRdrDC2000920063_en_US.exe is based on Installshield not inno?
![image](https://user-images.githubusercontent.com/40438607/82616080-c4902180-9bfe-11ea-8d2e-3d4eea46c0db.png)
Or this something wrong with my understanding between Installshield and inno?